Makefile

```diff
@@ -119,10 +119,34 @@ DTC		=	dtc
 # Guess the compillers xlen
 OPENSBI_CC_XLEN = `expr substr \`$(CC) -dumpmachine\`  6 2`
 
+# Setup platform XLEN, ABI, ISA and Code Model
+ifndef PLATFORM_RISCV_XLEN
+ifeq ($(OPENSBI_CC_XLEN), 32)
+PLATFORM_RISCV_XLEN = 32
+else
+PLATFORM_RISCV_XLEN = 64
+endif
+endif
```

Comment:
This is hard to read. I think there are better ways to write it, but I don't know make quite well enough for it. But I do know it's possible to indent it so that the sequence and nesting of the if/else can be easier to see. Mind doing that, please?

Reply:
My bad for not indenting this properly. I will update this.

I could have certainly done "PLATFORM_RISCV_XLEN = $(OPENSBI_CC_XLEN)" here but these makes output of (i.e. "V=1" command-line parameter) less readable so I used a if-else-ladder to ensure that verbose make output is still readable.